plugins/catalog-import/src/components/EntityListComponent/EntityListComponent.tsx

```diff
@@ -140,7 +153,13 @@ export const EntityListComponent = (props: EntityListComponentProps) => {
                       : {})}
                   >
                     <ListItemIcon>{Icon && <Icon />}</ListItemIcon>
-                    <ListItemText primary={humanizeEntityRef(entity)} />
+                    <ListItemText
+                      primary={
+                        entityPresentationApi.forEntity(
+                          stringifyEntityRef(entity),
+                        ).snapshot.entityRef
+                      }
```

Comment:
```suggestion
                      primary={<EntityDisplayName entityRef={entity} />}
```

this really should work here unless i'm severely mistaken

Reply:
yeah sorry, at the time I was doing this changes I didn't knew about hide Icon in <EntityDisplayName> so directly using that was changing UI. But now I will change it to <EntityDisplayName> hide icon prop.